lib/tests/function.rs

```diff
@@ -200,6 +200,38 @@ async fn function_array_append() -> Result<(), Error> {
 	Ok(())
 }
 
+#[tokio::test]
+async fn function_array_at() -> Result<(), Error> {
+	let sql = r#"
+		RETURN array::at(["hello", "world"], 0);
+		RETURN array::at(["hello", "world"], 3);
+		RETURN array::at(["hello", "world"], -1);
+		RETURN array::at(["hello", "world"], -3);
```

Comment:
@hchockarprasad could we add test statements for also checking what happens with an empty array and `0`, `3`, `-3` indexes?

Reply:
Yea..Have added few tests to check for empty array as well.